modules/javafx.base/src/main/java/com/sun/javafx/binding/ListenerManager.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.sun.javafx.binding;
+
+import java.util.Objects;
+
+import javafx.beans.InvalidationListener;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+
+/**
+ * Manages a single data field of type {@link Object} to store zero,
+ * one or more {@link InvalidationListener}s and {@link ChangeListener}s. This
+ * helps to minimize the storage requirements for keeping track of these
+ * listeners.<p>
+ *
+ * When there are no listeners, the field will be {@code null}. When there is
+ * only a single invalidation listener or change listener, the field will contain
+ * only that listener. When there is more than one listener, the field will hold
+ * a {@link ListenerList}. It is recommended to never inspect this field directly
+ * but always use this manager to interact with it.
+ *
```

Comment:
I suggest adding a line that says that this class is used by bindings.

Reply:
I think we shouldn't document obvious things as that's very little added value. Any IDE can show you where the class is used.